extras/ci/ci-queue-status.py

```
@@ -66,12 +66,15 @@ def load_label_groups():
             name = group.get("name", "Other")
             self_hosted = bool(group.get("self_hosted", False))
             groups.append((labels, name, self_hosted))
-        return groups or DEFAULT_LABEL_GROUPS
+        prefixes = []
+        for p in config.get("runner_name_prefixes", []):
+            prefixes.append((p.get("prefix", ""), p.get("name", "Other"), bool(p.get("self_hosted", False))))
+        return groups or DEFAULT_LABEL_GROUPS, prefixes
```

Ruff (0.15.2)

[warning] Consider moving this statement to an `else` block

(TRY300)